src/app/global-header/global-header.component.scss

```diff
@@ -1,15 +1,15 @@
 header {
-  display: flex;
-  justify-content: space-between;
-  flex-direction: row;
-  align-items: center;
-  gap: 1rem;
+  // display: flex;
+  // justify-content: space-between;
+  // flex-direction: row;
```

Comment:
@Ekta29yadav Please delete the code if not required

Reply:
@itsvick Done